x-pack/plugins/observability/public/pages/alerts/components/alert_actions.tsx

```diff
@@ -68,32 +51,36 @@ export function AlertActions({
     http: {
       basePath: { prepend },
     },
+    triggersActionsUi,
   } = useKibana().services;
   const userCasesPermissions = useGetUserCasesPermissions();
-  const { mutateAsync: untrackAlerts } = useBulkUntrackAlerts();
+
+  const data = useMemo(
```

Comment:
Is this conversion really necessary? I see we use it in `getRuleIdFromEvent`, and I wonder if we can change `getRuleIdFromEvent` in order to avoid this conversion.

Reply:
As discussed separately, this needs a bit more reasoning and discussion with the involved apps and thus will be covered in a separate PR